src/family/family.service.ts

```diff
@@ -1,12 +1,21 @@
 import { Injectable } from '@nestjs/common';
+import type { Family } from '@prisma/client';
+
+import { PrismaService } from '../prisma/prisma.service';
 
 @Injectable()
 export class FamilyService {
+  constructor(private readonly prisma: PrismaService) {}
+
   findAll() {
     return `This action returns a families`;
   }
 
-  findOne(id: number) {
-    return `This action returns a #${id} family`;
+  findFamilyById(id: string): Promise<Family | null> {
```

Comment:
isn't better to name this function "findById" ? because it's inside the FamilyService and we are duplicating the context.

https://github.com/kettanaito/naming-cheatsheet#avoid-context-duplication

Reply:
Yep, You're right.